intrange.go

```diff
@@ -539,13 +541,21 @@ func compareNumberLit(exp ast.Expr, val int) bool {
 	}
 }
 
-func castNStr(pass *analysis.Pass, i *ast.Ident, n string) string {
-	initType := pass.TypesInfo.TypeOf(i).String()
-	if initType == "int" {
-		return n
+func operandToString(pass *analysis.Pass, i *ast.Ident, operand ast.Expr) string {
+	s := recursiveOperandToString(operand)
+	t := pass.TypesInfo.TypeOf(i)
+
+	if t == types.Typ[types.Int] {
+		if len(s) > 5 && s[:4] == "int(" && s[len(s)-1:] == ")" {
```

Comment:
gt 5 is correct here. 4 (`len("int(")`) + 1 or more (`len(args)`) +1 (`len(")")`). If the string is not 6 or more we do not want to go into this condition. Further the out of range comment doesn't make sense because 5 is greater than 4.